docs/content.md

```diff
@@ -1,86 +1,82 @@
-# Design Guidelines content & IA
+# Interface Guidelines: proposed IA
 
 ## Guidelines
 
-###  Global
+###  Principles
 
-* Design Principles
-* Accessibility
-* Voice & tone (off link to Manual of Style?)
-* Designing for Safety
+* Accessibility: (Inclusive) consider a11y in design early, list of tools
```

Comment:
Should this just be "Inclusivity": ? Confused about why inclusive is in ()

Reply:
I'm not sure the term "Inclusivity" is as widely used as "Inclusive", but I agree that we should try to call it "inclusive design" (once we find the correct/best term).